src/java.base/share/classes/jdk/internal/loader/NativeLibraries.java

```diff
@@ -54,14 +61,20 @@
  * will fail.
  */
 public final class NativeLibraries {
-    private static final boolean loadLibraryOnlyIfPresent = ClassLoaderHelper.loadLibraryOnlyIfPresent();
+    private static boolean loadLibraryOnlyIfPresent;
     private final Map<String, NativeLibraryImpl> libraries = new ConcurrentHashMap<>();
     private final ClassLoader loader;
     // caller, if non-null, is the fromClass parameter for NativeLibraries::loadLibrary
     // unless specified
     private final Class<?> caller;      // may be null
     private final boolean searchJavaLibraryPath;
 
+    private static boolean loadLibraryOnlyIfPresentInitialized;
+    private static void initLoadLibraryOnlyIfPresent() {
+        loadLibraryOnlyIfPresent = ClassLoaderHelper.loadLibraryOnlyIfPresent();
+        loadLibraryOnlyIfPresentInitialized = true;
+    }
```

Comment:
Please explain the motivation for this change relative to upstream.

Reply:
Added a comment and a link to this PR.

Reply:
A link to this PR isn't terribly helpful. Please expand on the "bootstrap issue" that is being avoided; ideally with a link to an issue with all the details. For the record, I haven't seen any issues without this change.

Reply:
This PR description has the `ExceptionInInitializerError`/`bootstrap error` stacktrace caused by the initialization in question.
It can be moved to an issue if a PR is not sufficient.

Reply:
I would favour removing this difference unless there's a clear explanation of why it's necessary. The description of this PR doesn't explain how that problem arises.

Reply:
I updated the PR description with following:
This was caused by [OpenJ9 jclcinit.c:initializeRequiredClasses() J9VMCONSTANTPOOL_JDKINTERNALLOADERNATIVELIBRARIES](https://github.com/eclipse-openj9/openj9/blob/fd647ffb085b200979db35430d664a0b773a5a08/runtime/jcl/common/jclcinit.c#L590)
```
initializeRequiredClasses(J9VMThread *vmThread, char* dllName)
{
	static UDATA requiredClasses[] = {
			J9VMCONSTANTPOOL_JAVALANGTHREAD,
...
			J9VMCONSTANTPOOL_JDKINTERNALLOADERNATIVELIBRARIES,
```

Reply:
I don't see any problems during a build; please explain how to reproduce the problem this is intended to address or revert this change.